gcc/config/arc64/arc64.h

```diff
@@ -506,6 +506,17 @@ extern const enum reg_class arc64_regno_to_regclass[];
        "jl\t[r12]\n"						\
        TEXT_SECTION_ASM_OP);
 
+/* ATOMIC options.  */
+/* FIXME: is 0 okay or should it be -1 like DEFAULT_arc_mpy_option?  */
+/* Default atomic option value.  */
+#undef DEFAULT_ARC_ATOMIC_OPTION
+#define DEFAULT_ARC_ATOMIC_OPTION 0
+
+#define TARGET_ATOMIC_1 (arc_atomic_option > 0)
```

Comment:
Please use `ARC64_HAS_ATOMIC_1` macro name. Likewise for all atomic options fallowing.